File: backend/services/ControlService/Models/Account.cs
namespace ControlService.Models
{
    public class Account
    {
        public int Id { get; set; }
        public string? Username { get; set; }
        public string? Password { get; set; }
        public string? PhoneNumber { set; get; }
        public string? Info {  get; set; }
        public byte[]? Icon { get; set; }
    }
}

File: frontend/src/models/account.ts
export interface Account {
    id: number;
    userName: string;
    password: string;
    phoneNumber: string;
}

export interface AddAccountDto {
    userName: string;
    password: string;
    phoneNumber: string;
}

export interface LoginAccountDto {
    Username: string;
    Password: string;
    PhoneNumber: string;
}
